### PR TITLE
Deploy fluentbit to aks-secure-baseline spike infrastructure

### PR DESCRIPTION
**Pull request by AAkindele**
# Type of PR

- [x] Documentation changes
- [ ] Code changes
- [ ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## Purpose of PR

Deploy fluentbit with GitOps in infrastructure created from the aks-secure-baseline spike.

The goals are to document
- steps needed to get this working in the aks-secure-baseline infrastructure
- document other information learned during the spike
- document open questions that still need to be answered after the spike

## Validation

- [ ] Unit tests updated and ran successfully
- [x] Update documentation or issue referenced above

fluentbit-logs-to-log-analytics

## Issues Closed or Referenced

- Closes https://github.com/retaildevcrews/ngsa/issues/521

### CODE REVIEW

File: spikes/aks-secure-baseline/deployments/spike/east/fluentbit.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
```
Comment (AAkindele) [inline spikes/aks-secure-baseline/deployments/spike/east/fluentbit.yaml:1]: This file was copied from fluenbit helm chart in ngsa and modified.
- chart definition was updated to point to this spike
- container image was updated to pull from private container registry from aks-secure-baseline spike

File: spikes/aks-secure-baseline/README.md [hunk 2]
```diff
@@ -138,3 +141,24 @@ az acr import --source ghcr.io/retaildevcrews/ngsa-app:beta -n $ACR_NAME
 az acr import --source ghcr.io/retaildevcrews/ngsa-lr:beta -n $ACR_NAME
 
 ```
+
+Fluentbit setup
```
Comment (alkofahi) [inline spikes/aks-secure-baseline/README.md:145]: nit: why not format this as a sub-header?

File: spikes/aks-secure-baseline/README.md [hunk 3]
```diff
@@ -138,3 +141,24 @@ az acr import --source ghcr.io/retaildevcrews/ngsa-app:beta -n $ACR_NAME
...
+
+```bash
+
+# create fluentbit namespace
+kubectl create namespace fluentbit
+
+Ngsa_Log_Analytics_RG="rg-enterprise-networking-hubs"
+Ngsa_Log_Analytics_Name="la-hub-eastus2-lq7hlzxsovd4c"
```
Comment (alkofahi) [inline spikes/aks-secure-baseline/README.md:153]: nit: I would advise against storing these values, how about using placeholders (i.e., <<your-rg-name>>, <<log-analytics-instance-name>>) and leave it to whoever wants to use this script to plug their values in?

**Review by alkofahi — COMMENTED** [review]
The PR type indicates documentation but I see changes to script files, will these changes affect existing deployment/setup scripts? If so, did we confirm that these scripts are working? Apologies if I am lacking context :)

**Comment by AAkindele** [comment]
> The PR type indicates documentation but I see changes to script files, will these changes affect existing deployment/setup scripts? If so, did we confirm that these scripts are working? Apologies if I am lacking context :)

@alkofahi
These changes are isolated in the `spikes` directory, So, they won't affect the existing deployment instructions. The commands here are documentation for steps taken during the spike to get the task working. The goal is to use this information after the spike is complete when updating/creating the real environment, scripts, automation, etc.

I just added a screenshot of the fluentbit logs in log analytics.

**Review by alkofahi — APPROVED** [review]

**Pull request merged**